test/Microsoft.Extensions.Localization.Tests/LocalizationServiceCollectionExtensionsTest.cs

```diff
@@ -21,15 +21,23 @@ public void AddLocalization_AddsNeededServices()
 
             // Assert
             var services = collection.ToList();
-            Assert.Equal(2, services.Count);
+            Assert.Equal(4, services.Count);
 
-            Assert.Equal(typeof(IStringLocalizerFactory), services[0].ServiceType);
-            Assert.Equal(typeof(ResourceManagerStringLocalizerFactory), services[0].ImplementationType);
+            Assert.Equal(typeof(IOptions<>), services[0].ServiceType);
```

Comment:
Ideally I think we do not want to test the services added by options itself as it makes test easy to break if Options decides to change the services (ex: adds/deletes a service etc.). I think we should just check for the services that localization adds itself. (ex: https://github.com/aspnet/Localization/pull/221)